_topic_maps/_topic_map.yml

```diff
@@ -1525,22 +1525,38 @@ Topics:
   Topics:
   - Name: Understanding multiple networks
     File: understanding-multiple-networks
-  - Name: Understanding user defined networks
-    File: understanding-user-defined-network
-  - Name: Configuring an additional network
-    File: configuring-additional-network
+  - Name: Primary networks
+    Dir: primary_networks
+    Distros: openshift-enterprise, openshift-origin
+    Topics:
+    - Name: UserDefinedNetwork CR
```

Comment:
I am OK with the new sections, in future we should be able to add similar-ish sections for the existing features that will be supported on UDNs or link to the original docs for those (policies, routes, eips, svc etc)